src/Metric.php

```diff
@@ -92,6 +101,38 @@ public function scan($uri, \DOMXPath $xpath, $depth, Page $page, Metrics $contex
     }
 
     /**
+     * Get the machine name for a given error message
+     * 
+     * @param $error_message
+     * @return string
+     */
+    public function getMarkNameFromMessage($error_message)
+    {
+        switch ($error_message) {
```

Comment:
Personally, I prefer using if/else when you're wanting to do sequential evaluation of expressions, I think it helps to understand the flow of the logic. However, I understand that switches like this tend to be more readable when many cases are added and the logic still flows top-to-bottom.